lib/python/treadmill_aws/cli/admin/aws/instance.py

```diff
@@ -118,11 +118,16 @@ def configure(instance):
         multiple=True,
         help='IPA hostgroup memberships',
     )
+    @click.option(
+        '--hostname',
+        multiple=True,
```

Comment:
why multiple?

Reply:
My thought was to allow one-command provisioning of service clusters; i.e. --hostname zk1.foo.com --hostname zk2.foo.com --hostname zk3.foo.com. They should all share the same environment variables passed by --data, so why issue the command three times?